include/swift/Option/Options.td

```diff
@@ -519,6 +519,14 @@ def enable_experimental_concise_pound_file : Flag<["-"],
   Flags<[FrontendOption]>,
   HelpText<"Enable experimental concise '#file' identifier">;
 
+def enable_experimental_cxx_interop :
+  Flag<["-"], "enable-experimental-cxx-interop">,
+  HelpText<"Enable C++ interop code generation and config directives">;
```

Comment:
```suggestion
  HelpText<"Allow importing C++ modules into Swift (experimental feature)">;
```

I'm not sure what config directives are.

Reply:
I copied the `HelpText` from the frontend flag `-enable-cxx-interop` -- but I like your wording better. Done.